types/ml-levenberg-marquardt/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for ml-levenberg-marquardt 1.0
+// Type definitions for ml-levenberg-marquardt 3.1
 // Project: https://github.com/mljs/levenberg-marquardt#readme
 // Definitions by: m93a <https://github.com/m93a>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
```suggestion
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
please drop on next line:
```ts
// TypeScript Version: 2.1
```

Reply:
Removed in https://github.com/mvernacc/DefinitelyTyped/commit/e0d2fbe8672728c750ce4f556fe6aee34c5b4d22